src/Schema/Attribute/Attribute.php

```diff
@@ -6,69 +6,124 @@
 
 class Attribute extends Item implements AttributeSingle
 {
+    /**
+     * @var static|null
```

Comment:
here is `string` instead of `static|null`

Reply:
static|null is probably an error;, but string|null is used because the property is instantiated empty- if it's string for all intents and purposes, then it'll probably just require a string typecast here and there on methods that *may* return null.

Reply:
`string|null` is fine for me (for all the other similar comments too)

Reply:
pretty sure I've got these now; will need to review & double-check on the php-7.1 branch later.